src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => (a.toString() > b.toString() ? 1 : -1),
+  ) {
```

Comment:
The default compare function uses a lexicographical comparison which is fine for strings. However, for numbers, this might not work as expected since '2' will be greater than '10'. Consider using a numerical comparison when the array elements are numbers.